morpheus/_lib/src/stages/write_to_file.cpp

```diff
@@ -90,11 +97,12 @@ WriteToFileStage::operator_fn_t WriteToFileStage::build_operator()
 }
 
 // ************ WriteToFileStageInterfaceProxy ************* //
-std::shared_ptr<WriteToFileStage> WriteToFileStageInterfaceProxy::init(neo::Segment &parent,
-                                                                       const std::string &name,
-                                                                       const std::string &filename,
-                                                                       const std::string &mode,
-                                                                       FileTypes file_type)
+std::shared_ptr<srf::segment::Object<WriteToFileStage>> WriteToFileStageInterfaceProxy::init(
+    srf::segment::Builder &parent,
```

Comment:
Same here, `parent` is kind of confusing, the Builder definitely isn't a parent of the Object.